bootstrap/main.go

```diff
@@ -56,6 +56,9 @@ func mainImpl() {
 		"weave_cloud_hostname": opts.WCHostname,
 	})
 
+	// Due to some users Kubernetes clusters having unvalid certificates,
+	// we default to skipping the certificate validation.
+	otherArgs = append(otherArgs, "--insecure-skip-tls-verify=true")
```

Comment:
Personally, I'd drop the `=true`.

Also, what's the flag parsing semantics when the flag is specified more than once? I wonder whether *pre*pending would allow the user to override the setting by supplying their own `--inscure-skip-tls-verify=false` on the command line.

Reply:
> Personally, I'd drop the =true.

Only wanted to make it clear, but have no strong preference, can skip it.

> Also, what's the flag parsing semantics when the flag is specified more than once?

I tested that, and you can pass it as often as you like, until you pass in false, which will be always override the `true` ones you pass. So I don't think that will be a problem.